go/oasis-test-runner/oasis/fixture.go

```diff
@@ -296,6 +316,9 @@ type ByzantineFixture struct {
 	Script       string `json:"script"`
 	IdentitySeed string `json:"identity_seed"`
 	Entity       int    `json:"entity"`
+
+	EnableDefaultLogWatcherHandlerFactories bool                        `json:"enable_default_log_fac"`
```

Comment:
Why does the byzantine node fixture have this while others do not?

Reply:
Other fixtures could have `DisableDefaultLogWatcherHandlerFactories bool` but I didn't include it everywhere to prevent "spam" the code too much since none of the current tests need that functionality.

It is trivial to add such a field when a node type would need it.

Reply:
Ok.